inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/audit/AuditRunnable.java

```diff
@@ -0,0 +1,185 @@
+            Boolean listAll) {
+        this.request = request;
+        this.auditId = auditId;
+        this.auditName = auditName;
+        this.auditVOList = auditVOList;
+        this.latch = latch;
+        this.restTemplate = restTemplate;
+        this.auditUrl = auditUrl;
+        this.auditIdMap = auditIdMap == null ? new HashMap<>() : auditIdMap;
+        this.listAll = listAll;
+    }
+
+    @Override
+    public void run() {
+        try {
+            List<AuditInfo> auditSet;
+            if (listAll) {
+                auditSet = getAuditInfoListByIp(request, request.getInlongGroupId(), request.getInlongStreamId(),
+                        request.getIp(), auditId);
+            } else {
+                auditSet = getAuditInfoList(request, request.getInlongGroupId(), request.getInlongStreamId(), auditId);
+            }
+            auditVOList.add(new AuditVO(auditId, auditName, auditSet, auditIdMap.getOrDefault(auditId, null)));
+            this.latch.countDown();
+        } catch (Exception e) {
+            LOGGER.error("query audit failed for request={}", request);
+            this.latch.countDown();
+            throw new BusinessException("query audit failed");
+        }
+
+    }
+
+    private List<AuditInfo> getAuditInfoListByIp(AuditRequest request, String groupId, String streamId, String ip,
+            String auditId) {
+        List<AuditInfo> auditSet = new ArrayList<>();
+        try {
+            String start = request.getStartDate();
+            String end = request.getEndDate();
+            if (StringUtils.isBlank(request.getEndDate())) {
+                end = SECOND_DATE_FORMATTER.parseDateTime(request.getEndDate()).plusDays(1).toString(SECOND_FORMAT);
+            }
+            StringBuilder builder = new StringBuilder();
+            builder.append(auditUrl);
+            if (StringUtils.isNotBlank(ip)) {
+                builder.append("/audit/query/getIps?")
+                        .append("&ip=").append(ip);
+            } else {
+                builder.append("/audit/query/minutes?")
+                        .append("&inlongGroupId=").append(groupId)
+                        .append("&inlongStreamId=").append(streamId);
+            }
+            builder.append("startTime=").append(start)
+                    .append("&endTime=").append(end)
+                    .append("&auditId=").append(auditId);
+            String url = builder.toString();
+            LOGGER.info("query audit url ={}", url);
+            AuditResponse result = HttpUtils.request(restTemplate,
+                    url,
+                    HttpMethod.GET, null,
+                    null,
+                    AuditResponse.class);
+            LOGGER.info("success to query audit info result={}", result);
+            return CommonBeanUtils.copyListProperties(result.getData(), AuditInfo::new);
+        } catch (Exception e) {
+            LOGGER.info("query audit failed for groupId={}, streamId={}", groupId, streamId, e);
```

Comment:
Better to output more information in log like `ip` and `auditId`

Reply:
Done.